tests/draft2019-09/optional/format/ipv6.json

```diff
@@ -62,6 +62,56 @@
                 "description": "mixed format with ipv4 section with a hex octet",
                 "data": "1::2:192.168.ff.1",
                 "valid": false
+            },
+            {
+                "description": "ipv4-mapped ipv6 address",
+                "data": "::ffff:192.168.0.1",
```

Comment:
This isn't already covered by `"mixed format with the ipv4 section as decimal octets"`?

Reply:
not quite.. I'll amend the test description.